src/_locales/en/messages.cson

```diff
@@ -79,31 +79,31 @@ downloadWith:
 unsupportedURL:
 	message: 'Unsupported download address: $1'
 confirmPauseIsStop:
-	message: 'This download cannot be resumed. Stop it?'
+	message: 'This download cannot be resumed. Abort?'
 confirmDeleteFile:
 	message: 'Delete downloaded file?'
 confirmRemove:
-	message: 'Remove this unfinished download?'
+	message: 'Remove current unfinished download?'
 general:
 	message: 'General'
 saveFileTo:
 	message: 'Save files to:'
 downloadFolder:
-	message: 'downloads folder'
+	message: 'Download folder'
 alwaysAsk:
-	message: 'always ask'
+	message: 'Always ask?'
 skipFirstSavingAttempt:
-	message: 'The first saving attempt is problematic and I want to skip it.'
+	message: 'The first saving attempt was problematic and was skipped'
```

Comment:
This is a checkbox option to workaround a [problem](https://github.com/jingyu9575/multithreaded-download-manager/wiki/Problems-with-disabling-the-feature-%22out-of-process-extensions%22) in some systems. When the problem happens, the first attempt to save will fail and the extension will then use an alternative slow method. Checking this option makes the extension skip the first attempt and use the alternative method directly.

Could you edit the translations to help clarifiy these items?

![image](https://user-images.githubusercontent.com/8923139/36478225-57842f84-1747-11e8-94c7-25c479b92e09.png)

Reply:
`and should be skipped`